runtime/src/main/java/io/quarkiverse/resteasy/problem/ProblemRuntimeConfig.java

```diff
@@ -6,8 +6,12 @@
 import io.smallrye.config.WithDefault;
 import io.smallrye.config.WithName;
 
+/**
+ * @deprecated v4.0.0 all configuration will be evaluated in build time
```

Comment:
Consider suplementing this with link to new version of _config_, sth like:
```java
/**
* ...
* <p> Use {@link ProblemBuildConfig} instead.
*/
```

Reply:
Link added